src/ui/components/TransferGrain.js

```diff
@@ -18,90 +17,151 @@ export type Props = {|
+  },
+  dropdown: {
+    width: "300px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    borderRadius: "500px",
+    background: "#F2F5F7",
+  },
+  centerRow: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  third: {
+    width: "300px",
+    height: "50px",
+    margin: "30px 15px",
+  },
+  half: {
+    width: "450px",
+    height: "50px",
+    margin: "30px",
+  },
+  memo: {
+    width: "960px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    paddingLeft: "20px",
+    color: "black",
+  },
+});
+
 export const TransferGrain = ({ledger, setLedger}: Props) => {
-  const [sourceIdentity, setSourceIdentity] = useState<Identity | null>(null);
-  const [destIdentity, setDestIdentity] = useState<Identity | null>(null);
+  const classes = useStyles();
+  const [sender, setSender] = useState<Account | null>(null);
+  const [receiver, setReceiver] = useState<Account | null>(null);
   const [amount, setAmount] = useState<string>("0");
-  const [maxAmount, setMaxAmount] = useState<Grain>(fromInteger(0));
   const [memo, setMemo] = useState<string>("");
 
-  const setSender = (acct: Account | null) => {
-    setMaxAmount(acct ? acct.balance : fromInteger(0));
-    setSourceIdentity(acct ? acct.identity : null);
-  };
-
-  const setReceiver = (acct: Account | null) => {
-    setDestIdentity(acct ? acct.identity : null);
-  };
-
-  const submitTransfer = (e) => {
-    e.preventDefault();
-    if (sourceIdentity && destIdentity) {
+  const submitTransfer = () => {
+    if (sender && receiver) {
       const nextLedger = ledger.transferGrain({
-        from: sourceIdentity.id,
-        to: destIdentity.id,
+        from: sender.identity.id,
+        to: receiver.identity.id,
         amount: fromFloatString(amount),
         memo: memo,
       });
       setLedger(nextLedger);
       setAmount(fromInteger(0));
-      setSender(nextLedger.account(sourceIdentity.id));
+      setSender(nextLedger.account(sender.identity.id));
+      setReceiver(nextLedger.account(receiver.identity.id));
       setMemo("");
     }
   };
 
   return (
-    <div
-      style={{
-        width: "80%",
-        margin: "0 auto",
-        background: "white",
-        padding: "0 5em 5em",
-      }}
-    >
-      <h1>Transfer Grain</h1>
-      <form onSubmit={(e) => submitTransfer(e)}>
-        <label>From</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setSender} />
-        <br />
-        <label>To</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setReceiver} />
-        <p>
-          <label htmlFor="amount">Amount</label> <br />
-          <input
-            type="number"
-            name="amount"
-            min="0"
-            step="any"
-            placeholder={`max: ${maxAmount}`}
-            required
-            value={amount}
-            onChange={(e) => setAmount(e.currentTarget.value)}
+    <div className={classes.root}>
```

Comment:
Can just use the Container component from MUI: https://material-ui.com/components/container/

If you need to set a maxWidth or something, you can do so from that components props. That way the look of this page can follow the theme instead of having a white BG

Reply:
I wanted the background to be white though, I was basing this UI off of Faruk's whimsical mockup: https://whimsical.com/A6EEdJgLwX3K4VTroURP3b@2Ux7TurymMnWnyv6hKCm. 
If the BG isn't white everything else looks very strange and the colors all have to change.